pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -58,3 +62,24 @@ func PersistentVolumeClaimUsageFunc(object runtime.Object) api.ResourceList {
 	}
 	return result
 }
+
+// PersistentVolumeClaimConstraintsFunc verifies that all required resources are present on the claim
+// In addition, it validates that the resources are valid (i.e. requests < limits)
```

Comment:
I'm not seeing where this is happening.  Isn't the usage func just a hardcoded count?